### DISCUSSION TITLE
How to save the image of a custom ColorChecker?

### DISCUSSION THREAD

**Discussion by xeggbee** [colour-science/colour#1102] [Q&A]
Is there a method to save the custom colorchecker?

After read the spectral distribution of colorchecker, i`d like to save the colorchecker to a file, like when we see it in the 
plot_single_colour_checker("ColorChecker 2005")  function.

![image](https://user-images.githubusercontent.com/39626787/218363652-44839e50-0748-49cf-b0b0-92010bd033bc.png)

**Comment by thomasmansencal** [comment]
Hello @xeggbee,

Try passing the `filename` kwarg as follows: `filename="my_file.png"`.

Cheers,

Thomas
